startup/00-startup.py

```diff
@@ -37,3 +37,17 @@ def show_env():
     a = out.decode('utf-8')
     b = a.split('\n')
     print(b[0].split('/')[-1][:-1])
+
+
+
+from databroker.assets.handlers import AreaDetectorTiffHandler
+
+class SaferAreaDetectorTiffHandler(AreaDetectorTiffHandler):
+    def __call__(self, *args, **kwargs):
+        try:
+            return super().__call__(*args, **kwargs)
+        except FileNotFoundError:
+            self._fpp = 1
```

Comment:
I do not understand this change.  If we are outputting documents that have the wrong frames per point, then we need to fix the document generation.

Reply:
@mrakitin made this and I don't know any other solution.

Reply:
Correction: This is "based on" solution by @mrakitin and I don't know any other solution.

Reply:
We ended up with this solution with @danielballan on Friday as a workaround to the scans which have broken documents.

Agree that we should fix the document generation.